Makefile

```diff
@@ -19,11 +19,11 @@ build:
 	poetry build
 
 .PHONY: code
-code: check format lint sort bandit test
+code: check format lint sort bandit test build
```

Comment:
I don't think that build should be part of the "code" target. The purpose of it was just to see if the code is properly formatted and linted while writing new code. I would not expect artifacts to be created when executing this target.